versioned_docs/version-1.1.x/tutorials/working-with-surrealdb-over-http-via-postman.mdx

```diff
@@ -0,0 +1,208 @@
+---
+sidebar_position: 2
+---
+
+# Working with SurrealDB over HTTP via Postman
+
+# Working with SurrealDB over HTTP via Postman
+
+SurrealDB provides a RESTful HTTP API for interacting with the database. This API is designed to be simple and easy to use, and is the primary way to interact with SurrealDB.
+
+In this tutorial, you will learn how to query SurrealDB endpoints via the [Postman collection](https://postman.com/surrealdb/workspace/surrealdb/collection/19100500-3da237f3-588b-4252-8882-6d487c11116a). SurrealDB supports requests via HTTP & Rest with which you can handle different queries from creating simple tables to having Graph relationships between complex tables.
+
+Check out the [HTTP & Rest integration documentation](/docs/integration/http) for a detailed list of all the endpoints supported.
+
+
+## Prerequisites
+
+This tutorial assumes that you have the following: 
+
+- A Postman account to fork the [SurrealDB collection](https://postman.com/surrealdb/workspace/surrealdb/collection/19100500-3da237f3-588b-4252-8882-6d487c11116a)
+- SurrealDB installed - If you do not, see the [installation guide](/docs/installation/overview) specific to your machine.
+
+## Getting started.
+
+Before heading forking the Collection from Postman, ensure that you have your SurrealDB instance running locally because by default the collection endpoint is set to [`localhost:8000`](http://localhost:8000) or [`http://127.0.0.1:8000`](http://127.0.0.1:8000/)
```

Comment:
```suggestion
Before forking the Collection from Postman, ensure that you have your SurrealDB instance running locally because by default the collection endpoint is set to [`localhost:8000`](http://localhost:8000) or [`http://127.0.0.1:8000`](http://127.0.0.1:8000/)
```